mediawiki.py

```diff
@@ -11,9 +11,9 @@
 from pprint import pprint
 
 class Importer(object):
-    def __init__(self, api_url, http_user=None, http_pass="", wiki_user=None, wiki_pass="", verbose=False):
+    def __init__(self, api_url, http_user=None, http_pass="", wiki_user=None, wiki_pass="", wiki_domain="", verbose=False):
         self.verbose = verbose
-        self.mw = simplemediawiki.MediaWiki(api_url,http_user=http_user,http_password=http_pass)
+        self.mw = simplemediawiki.MediaWiki(api_url, http_user=http_user, http_password=http_pass, domain=wiki_domain)
```

Comment:
Oops yeah sorry about that. Thought you were referring to it failing when the simplemediawiki lib didn't have the domain capability. I'm fixing this up right now and try to make it actually pullable.
Sorry that I'm causing this extra work for you.